### ISSUE TITLE
ObservableMedia does not emit when largest viewport is active

### ISSUE THREAD

**Issue by TylerDev6** [angular/flex-layout#275]
This seems to be happening with custom breakpoints and I'm using 2.0.0-beta6, but it may still be happening with other instances as well.

Here's a plunker that shows the custom breakpoints. Scaling the app up and down shows that events are emitted to console for small and medium, but not the largest size.

https://plnkr.co/edit/WvZtQ024vu2R816EMPQD?p=preview

**Comment by ThomasBurleson** [comment]
@TylerDev6 - you non-overlapping ranges appear to be wrong (they are overlapping):

```js
[{
    alias: 'xs',
    mediaQuery: 'screen and (max-width: 767px)'
}, {
    alias: 'sm',
    mediaQuery: 'screen and (max-width: 767px)'
}, {
    alias: 'md',
    mediaQuery: 'screen and (min-width: 768px) and (max-width: 1023px)'
}, {
    alias: 'lg',
    mediaQuery: 'screen and (min-width: 1024px)'
}, {
    alias: 'xl',
    mediaQuery: 'screen and (min-width: 1024px)'
}]
```

**Comment by TylerDev6** [comment]
right, that was a problem I had encountered. As you can see, I only really need 3 breakpoints but the API indicates that I must provide all 5. Since I didn't want layout changes triggered by the default xs and xl breakpoints I overrode them.

**Comment by TylerDev6** [comment]
However, why would that actually cause the behaviour I'm seeing? xs / sm overlap but I still get emits for xs.... lg and xl overlap but I don't get emits for either.

**Comment by ThomasBurleson** [comment]
Related to #245

**Comment by ThomasBurleson** [comment]
Will be fixed with PR #313.  
> See working plunker:  https://plnkr.co/edit/yylQr2IdbGy2Yr00srrN?p=preview

**Comment by ThomasBurleson** [comment]
Fixed with SHA 00ac57a
